app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
Code Style: It's a good practice to use type hinting for all function parameters and the return type. For example, the `cafe` parameter could be annotated with the `Cafe` type.